README.md

```diff
@@ -83,6 +83,17 @@ Simply disable upside down for iPad and everything works like a charm ([[#78]](h
 ```ld: library not found for -lRCTOrientation-tvOS```
 Just remove it from linked libraries and frameworks
 
+4. For Windows, locking to an orientation will only work on devices in tablet mode.
```

Comment:
it's unclear to me whether this is a design decision on our part (and what the rationale for it is) or a limitation imposed on your code by some Windows API (which one?). Could you clarify?

Reply:
![image](https://user-images.githubusercontent.com/34109996/99864370-bd1ee600-2b57-11eb-8c01-d3f0834df0c9.png)